exchanges/bybit/ratelimit.go

```diff
@@ -66,69 +70,93 @@ const (
-		getAssetDepositSubMemberAddressEPL:       request.NewRateLimitWithWeight(time.Minute, 30, 1),
-		getWithdrawRecordsEPL:                    request.NewRateLimitWithWeight(time.Minute, 30, 1),
-		getAssetCoinInfoEPL:                      request.NewRateLimitWithWeight(time.Minute, 30, 1),
-		getExchangeOrderRecordEPL:                request.NewRateLimitWithWeight(time.Minute, 30, 1),
-		interTransferEPL:                         request.NewRateLimitWithWeight(time.Minute, 20, 1),
-		saveTransferSubMemberEPL:                 request.NewRateLimitWithWeight(time.Minute, 20, 1),
-		universalTransferEPL:                     request.NewRateLimitWithWeight(time.Second, 5, 5),
-		createWithdrawalEPL:                      request.NewWeightedRateLimitByDuration(time.Second),
-		cancelWithdrawalEPL:                      request.NewRateLimitWithWeight(time.Minute, 60, 1),
-		userCreateSubMemberEPL:                   request.NewRateLimitWithWeight(time.Second, 5, 5),
-		userCreateSubAPIKeyEPL:                   request.NewRateLimitWithWeight(time.Second, 5, 5),
-		userFrozenSubMemberEPL:                   request.NewRateLimitWithWeight(time.Second, 5, 5),
-		userUpdateAPIEPL:                         request.NewRateLimitWithWeight(time.Second, 5, 5),
-		userUpdateSubAPIEPL:                      request.NewRateLimitWithWeight(time.Second, 5, 5),
-		userDeleteAPIEPL:                         request.NewRateLimitWithWeight(time.Second, 5, 5),
-		userDeleteSubAPIEPL:                      request.NewRateLimitWithWeight(time.Second, 5, 5),
-		userQuerySubMembersEPL:                   request.NewRateLimitWithWeight(time.Second, 10, 10),
-		userQueryAPIEPL:                          request.NewRateLimitWithWeight(time.Second, 10, 10),
-		getSpotLeverageTokenOrderRecordsEPL:      request.NewRateLimitWithWeight(time.Second, 50, 50),
-		spotLeverageTokenPurchaseEPL:             request.NewRateLimitWithWeight(time.Second, 20, 20),
-		spotLeverTokenRedeemEPL:                  request.NewRateLimitWithWeight(time.Second, 20, 20),
-		getSpotCrossMarginTradeLoanInfoEPL:       request.NewRateLimitWithWeight(time.Second, 50, 50),
-		getSpotCrossMarginTradeAccountEPL:        request.NewRateLimitWithWeight(time.Second, 50, 50),
-		getSpotCrossMarginTradeOrdersEPL:         request.NewRateLimitWithWeight(time.Second, 50, 50),
-		getSpotCrossMarginTradeRepayHistoryEPL:   request.NewRateLimitWithWeight(time.Second, 50, 50),
-		spotCrossMarginTradeLoanEPL:              request.NewRateLimitWithWeight(time.Second, 20, 50),
-		spotCrossMarginTradeRepayEPL:             request.NewRateLimitWithWeight(time.Second, 20, 50),
-		spotCrossMarginTradeSwitchEPL:            request.NewRateLimitWithWeight(time.Second, 20, 50),
+var rateLimits = request.RateLimitDefinitions{
+	defaultEPL:                               request.NewRateLimitWithWeight(time.Second*5 /* See: https://bybit-exchange.github.io/docs/v5/rate-limit */, 600, 1),
```

Comment:
🚧 Definitely not okay to have a comment in the middle like this, IMO.
```suggestion
	defaultEPL:                               request.NewRateLimitWithWeight(time.Second*5, 600, 1),  // https://bybit-exchange.github.io/docs/v5/rate-limit
```